apps/mux/README_INTERNAL.md

```diff
@@ -0,0 +1,22 @@
+
+# Development
```

Comment:
This could be README since instructions/marketing text will live outside of Github.